community/community-leaders.md

```diff
@@ -62,6 +64,7 @@ Details on how to become a [Core Developer here](broken-reference).
 | [**Tomasz Kajetan Stańczak**](https://www.linkedin.com/in/tomaszkajetanstanczak/)             | Nethermind                          |
 | [**Anais Ofranc**](https://github.com/Consianimis)                                            | Consianimis                         |
 | [**Hamza Tokuchi**](https://github.com/Meuko)                                                 | [BlockLab](https://www.blocklab.nl) |
+| [**Ryan Fleischmann**](https://github.com/fleischr)                                           | Provide                             |
```

Comment:
I do believe this was only meant to be a list of core devs - not necessarily the maintainers.

Reply:
hm yea, its titled Maintainers though and as i said it seems quite outdated, lets wait for @GoldenBit0 input here

Reply:
@fleischr please, change `Core Developers Team` to `Maintainers Team` and update the list accordingly to accurately reflect the current maintainer list.

Rather than changing the list, please, add the Sentence `In September of 2022, Sonal Patel took over John Wolpert's TSC seat and role for the remainder of the 2021/2022 term.`

@GoldenBit0 @skosito @Kasshern @ognjenkurtic @Ybittan @Manik-Jain any objections to the amove?

Reply:
@Therecanbeonlyone1969 I agree with the suggested text regarding both the Core Devs>Maintainers and TSC seat change.

I don't think we should retain lists that need to be manually maintained, except for TSC and PGB which gets updated by Oasis/EEA admin.

All of these files within the [community folder](https://github.com/eea-oasis/baseline/tree/main/community) are really outdated, we went through an extensive exercise of updating the docs and fixing the synchronization between the [docs in the repo](https://github.com/eea-oasis/baseline/tree/main/docs) to the [gitbook docs]](https://docs.baseline-protocol.org/). 

@OASIS-OP-Admin please provide feedback on the following- I suggest this folder should be removed as the docs in the repo located here should be the single source of truth for this info, or can Oasis project support go through the docs located in the repo & update the content using the updated content in the gitbook docs?

Reply:
@GoldenBit0 @Therecanbeonlyone1969 Can it be assumed here the named reviewers of this PR are the most up to date list of maintainers? I want to make sure I'm providing a fully correct list.

Reply:
Yes that list is accurate.

Reply:
+1